### DISCUSSION TITLE
Trigger push notification on phone when car state is "Driving"

### DISCUSSION THREAD

**Discussion by RobertPaulson90** [teslamate-org/teslamate#1921] [Ideas]
Hey! I got a cool idea. I'd like to get a push notification my phone when my car goes into drive mode. I found an app called PushCut which lets me create a notification accompanied with a webhook URL to trigger it. Now I just need to link Teslamate and the postgresql "State" table somehow. Any suggestions where to implement this code?

**Comment by eliluong** [comment]
You can subscribe to the MQTT topic for driving state (https://docs.teslamate.org/docs/integrations/mqtt) and use whatever technique to monitor it and send a push notification.
